Documentation/GettingStarted.md

```diff
@@ -0,0 +1,41 @@
+# Getting Started with WheelMUD
+
+## Prerequisites
+Basically any modern Windows environment with any popular C# development environment, can be used.
+However, we currently recommend using Visual Studio 2013.
+For more details, check the [Basic Prerequisites](BasicPrerequisites.md) page.
+
+## Additional Requirements
+* .NET Framework 4 (http://www.microsoft.com/en-us/download/details.aspx?id=17718).
+* Any Telnet client, such as:
+ * [zMUD or cMUD](http://www.zuggsoft.com/index.php). Heavy, feature-rich telnet client, used by tons of serious MUD users.
+ * [PuTTY](http://www.putty.org). Lightweight telnet client, but remembers session info.
+ * [fTelnet / HtmlTerm](https://www.ftelnet.ca). Lightweight Flash and HTML5 based clients.
+ * Telnet.exe: Built into all supported versions of Windows, but the Telnet Client feature generally starts disabled. Open up the "Turn Windows features on and off" control panel, scroll down and ensure "Telnet Client" gets selected.
+
+## Getting the Source Code
+It is preferred to fork the code from [WheelMud/WheelMUD](https://github.com/WheelMud/WheelMUD).
+On the right side of github you will see the "HTTPS clone URL" which you can use with git to pull down a local repository.
+If you don't know how to work with git and github, you will want to follow a few tutorials on those tools first.
+
+## Building and Launching the Server
```

Comment:
I can't help but think we could have a binary release in order to protect people from repackagers